src/frontend/web_application/src/scenes/ContactBook/presenter.jsx

```diff
@@ -62,17 +59,14 @@ class ContactBook extends Component {
   constructor(props) {
     super(props);
     this.state = {
-      tags: [],
+      tags: [].concat(generateStateFromProps.bind(props)),
```

Comment:
@ziir is it the right way to set tags in constructor? (related to your comment in last PR)